vault/dendron.tutorial.md

```diff
@@ -2,28 +2,34 @@
 id: 678c77d9-ef2c-4537-97b5-64556d6337f1
 title: Getting Started
 desc: ''
-updated: 1631317031955
+updated: 1639532189097
 created: 1608051264282
 nav_order: 1.1
 ---
 
-
 ## Pre-requisites
 
-This walkthrough will help you get started with Dendron. Before you start, make sure you do the following:
+This walkthrough will help you get started with the basics of Dendron. Before you start, make sure you do the following:
+
 - installed [VSCode](https://code.visualstudio.com/download) or a VSCode compatible editor like [VSCodium](https://vscodium.com/)
-- installed [Dendron](https://marketplace.visualstudio.com/items?itemName=dendron.dendron) from the code marketplace
-- initialize your first Dendron workspace
+- installed [Dendron](https://link.dendron.so/vscode) from the code marketplace
+- initialize your first Dendron **[[workspace|dendron://dendron.dendron-site/dendron.topic.workspace]]**
+
+![[dendron://dendron.dendron-site/dendron.topic.workspace#summary,1:#*]]
 
-When you first install Dendron, you can click on the green `Initialize Workspace` button on the Welcome screen to initialize your first workspace. Then, pick a location to store your notes, and you'll be brought to the tutorial experience, which will contain notes with the same tutorial content here.
+![[dendron://dendron.dendron-site/asset.preview#tutorial-welcome-screen,1:#*]]
 
-![Welcome Screen](https://org-dendron-public-assets.s3.amazonaws.com/images/tutorial-welcome-screen.png)
+When you install Dendron, click the `Get Started` button to begin. You'll start with the same tutorial content in a new workspace created for you.
```

Comment:
can we move this ahead of the note reference? and update:

`You'll see the following page. Click ...`

Reply:
> You'll start with the same tutorial content in a new workspace created for you.

Not sure I understand this line. Can we cut?